cardano-foundation/cf-cardano-ballot
# feat(cip-1694-ui): inject env vars via script

**How to test locally**:

-  remove .env files if present (otherwise it would be taken as a source)
- run `docker build -t cip1694-ui:latest .`
- run `docker run -it --rm -e REACT_APP_EVENT_ID=WHATEVER1 -p 8080:8080 cip1694-ui:latest`, the `-e REACT_APP_blabla=http://shwatever/` allows you to pass env vars to the docker container
- open an app on `localhost:8080`
-  check network tab that should make post request with proper event id
